src/test/functional/steps/opening-hours.ts

```diff
@@ -0,0 +1,85 @@
+import { Then, When } from 'cucumber';
+import { expect } from 'chai';
+
+import * as I from '../utlis/puppeteer.util';
+
+When('I click the opening hours tab', async () => {
+  const selector = '#tab_opening-hours';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
```

Comment:
Can shorten to: 

`expect(await I.checkElement(selector)).equal(true);` if you want

Reply:
I generally think async calls look a bit better on their own line.